syntax = "proto3";

package proto;

option go_package = "./"; 



message EmbeddingsMessageRequest{
    string text = 1; 
}

message EmbeddingsMessageResponse{
    repeated float result = 1;

}

service EmbeddingService {

    rpc  TextToEmbedding(EmbeddingsMessageRequest) returns (EmbeddingsMessageResponse);
}